### PR TITLE
Restrict number of digits for pagination input

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Brief feature description
-Restrict pagination input box to accepts maximum of 5 digits.

### CODE REVIEW

File: app/javascript/search/pagination.js [hunk 1]
```diff
@@ -35,7 +35,7 @@ const Pagination = ({
       className='page_number'
       value={pageNumber}
       type='number'
```
Comment (Nasir2712) [inline app/javascript/search/pagination.js]: why can't you just simply do maxLength='5'?

Comment (sunilb0575) [reply app/javascript/search/pagination.js]: tried not working.

Comment (Nasir2712) [reply app/javascript/search/pagination.js]: how about just this``` if (event.target.value.length > 5) return false; ``` in the onchange method? also max length is only for type='text'.

Comment (ratneshraval) [reply app/javascript/search/pagination.js]: Let's try cleave.js and see if that helps.

**Pull request merged**